tests/test_virtualenv.py

```diff
@@ -332,3 +334,38 @@ def test_relative_symlink(tmpdir):
     shutil.rmtree(workdir)
 
     assert os.path.exists(lib64)
+
+
+def test_missing_certifi_pem(tmpdir):
+    """Make sure that we can still create virtual environment if pip is
+    patched to not use certifi's cacert.pem and the file is removed.
+    This can happen if pip is packaged by Linux distributions."""
+    tests_dir = os.path.dirname(__file__)
+    support_original = os.path.join(tests_dir, "../src/virtualenv_support/")
```

Comment:
this most certainly will fail on Windows; and indeed it has https://dev.azure.com/pypa/virtualenv/_build/results?buildId=3008 (not sure why the build does not show up here as Github check, although the job did start, you can see new jobs at https://dev.azure.com/pypa/virtualenv/_build?definitionId=11&_a=summary)

Reply:
Please also add a changelog entry about the change while you are at it.